internal/aez/power.go

```diff
@@ -5,20 +5,116 @@ import (
 	"io"
 )
 
+type pwr uint16
+
+func (data pwr) voltageADC() float64 {
+	return voltageConstant * float64(data)
+}
+
+type pwrt pwr
+
+func (data pwrt) voltage() float64 {
+	return pwr(data).voltageADC()
+}
+
+func (data pwrt) resistance() float64 {
+	return 3300/data.voltage() - 1000
```

Comment:
Same here with /. Should really be a linter thing though.

Reply:
3300 is really 3.3 * 1000, which to my mind looks clearer, since that is what the specs say. 

You could also perhaps make the base resistance an argument and use the same code for both NTC resistances, e.g.
`func (data pwrt) resistance() float64 { return resistance_ntc(1000) }`
and above
`func (data htrt) resistance() float64 { return resistance_ntc(3900) }`
but I have no idea if that is possible in `go`, or indeed if it would actually make anything much clearer since the code duplication is so small anyway.

Reply:
I considered such formats but since as you say the code duplication is small I wanted each formula as written in the spec in a single function, but I could do something about the `3.3*1000` (space probably won't be allowed)

Reply:
3.3e3? the closes you can get to the spec. 🤓

Reply:
That wold be acceptable.

Reply:
I did `3.3*1000` now and if anyone wants further changes they are free to make a pr :stuck_out_tongue_winking_eye:

Reply:
I think @local-minimum's solution is closer to the specs than 3.3e3. If you *really* want to get close to the specs, then go for 3.3*1e3, but 1000 is just as good to me.